### PR TITLE
Feature/docker

### PR DESCRIPTION
**Pull request by rpshep**
Tested. Compared results to file from manually run grass instead of grass as part of brass and results match exactly.

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/prerelease.sh
+/README.md
+/INSTALL
+/LICENCE.TXT
+/CHANGES.md
+/.gitignore
+/.git
+/docs.tar.gz
```
Comment (keiranmraine) [inline .dockerignore]: TL;DR - add a new line at the end of the file

Your editor needs setting to always include a new line at the end of a file.  Files are not technically text without this.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+RUN mkdir -p $OPT
+COPY --from=builder $OPT $OPT
+
+## USER CONFIGURATION
+RUN adduser --disabled-password --gecos '' ubuntu && chsh -s /bin/bash && mkdir -p /home/ubuntu
+
+USER    ubuntu
+WORKDIR /home/ubuntu
+
+CMD ["/bin/bash"]
```
Comment (keiranmraine) [inline Dockerfile]: Same new line issue

**Review by keiranmraine — CHANGES_REQUESTED** [review]
Couple of issues highlighted in line, plus:

## README.md

* Need to add link to quay.io page, see BRASS readme for example.
* Can you fix an errors in the markdown so heading actually displays as a heading (just needs space after hashes).
  * `Dependencies/Install`
  * `Options` section has several
* Update Licence statements with this year as the end date (add range if not already present) on any edited files
  * `README.md`
  * `brass_bedpe2vcf.pl`
  * `grass.pl`
  * `Makefile.PL`

File: .dockerignore (hunk 1 above)
Comment (rpshep) [reply .dockerignore]: Added all of the changes.

**Review by keiranmraine — APPROVED** [review]

**Pull request merged**